tests/test_designmatrix_noisepars.py

```diff
@@ -0,0 +1,62 @@
+import os
+import pint.models.model_builder as mb
+import pinttestdata
+import pytest
+
+
+@pytest.fixture()
+def model_and_toas():
+    datadir = pinttestdata.datadir
+    parfile = os.path.join(datadir, "J1713+0747_NANOGrav_11yv0_short.gls.par")
+    timfile = os.path.join(datadir, "J1713+0747_NANOGrav_11yv0_short.tim")
+    return mb.get_model_and_toas(parfile, timfile)
+
+
+def test_designmatrix_noise_exception_no_free_noise_param(model_and_toas):
```

Comment:
This name seems to suggest that it expects an exception?

Reply:
Yes.. That's confusing. I'll rename that.